src/pages/home/components/DisplaySubjects.jsx

```diff
@@ -4,7 +4,7 @@ import CardContent from "@material-ui/core/CardContent";
 import Card from "@material-ui/core/Card";
 import Typography from "@material-ui/core/Typography";
 
-import DisplaySubjectButton from "./DisplaySubjectButton";
+import DisplaySubjectButton from "./DIsplaySubjectButton";
```

Comment:
Why

Reply:
the file name has an upper case I

Reply:
![image](https://user-images.githubusercontent.com/41790090/78868256-dfd61180-7a42-11ea-8e65-e7dbc01ce852.png)
@mahmoudyusof